core/domain/rating_services_test.py

```diff
@@ -175,3 +178,30 @@ def test_invalid_exploration_ids_are_forbidden(self):
             ):
             rating_services.assign_rating_to_exploration(
                 self.USER_ID_1, 'invalid_id', 3)
+
+    def test_rating_assignation_with_no_exploration_summary_ratings(self):
+
+        def _mock_get_exploration_summary_by_id(exp_id):
+            """Assign None to exploration summary ratings."""
+            exp_summary_model = exp_models.ExpSummaryModel.get(exp_id)
+            if exp_summary_model:
```

Comment:
No complex logic in mocks (or in tests, in general). Just get the summary model, set its ratings and return that.

Reply:
Done